docs/ref/lib/experimental/pseudonymisation.rst

```diff
@@ -17,15 +17,24 @@ Summary
     Pseudonymisation provides a list of identifying keywords that includes UIDs
     and an anonymisation strategy that utilises SHA3_256 to hash text which is then
     encoded as base64, or for UIDs, converted to an integer appended to the PyMedPhys Org Root.  Dates are shifted consistently and ages are jittered.
-    The list of identifying keywords and the anonymisation strategy are passed in to
-    the (non-experimental) anonymisation module/functions.
+    The anonymisation strategy is passed in to the (non-experimental) anonymisation module/functions.
+    Anonymisation strategies are immutable maps that contain dispatch tables for replacing the values of identifying elements,
```

Comment:
Target audiences are programmers and non-programmers alike, only common denominator that can be assumed is that they are a Medical Physicist. Works like "immutable" "dispatch", "Tri-State" are all words I wouldn't expect a Medical Physicist to necessarily understand.

Might it be worth writing a doctest styled string (https://docs.python.org/3/library/doctest.html). For an example see:

https://github.com/pymedphys/pymedphys/blob/991dd077b6a6fa1e824d8490b5138025d492a0ad/pymedphys/_mudensity/mudensity.py#L88-L156

Which is then propagated into the docs over at:

https://docs.pymedphys.com/ref/lib/mudensity.html#pymedphys.mudensity.calculate

...also, of note, the following lines:

https://github.com/pymedphys/pymedphys/blob/2bf503be14227f5c460056428a65fdddc2ec01a5/.github/workflows/on-push.yml#L251-L256

Actually dives in and runs all of these examples to make sure that they work as written.

Reply:
I will provide more explanation of terminology, and take a look at the references